phys2bids/physio_obj.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+I/O objects for phys2bids.
+"""
+
+import numpy as np
+
+
+def is_valid(var, var_type, list_type=None):
+    """
+    Checks that the var is of a certain type.
+    If type is list and list_type is specified,
+    checks that the list contains list_type.
+    Input
+    -----
+    var: any type
+        Variable to be checked.
+    var_type: type
+        Type the variable is assumed to be.
+    list_type: type
+        Like var_type, but applies to list elements.
+
+    Output
+    ------
+    var: any type
+        Variable to be checked (same as input).
+    """
+    if not isinstance(var, var_type):
+        raise AttributeError(f'The given variable is not a {var_type}')
+
+    if var_type is list and list_type is not None:
+        for element in var:
+            _ = is_valid(element, list_type, return_var=False)
+
+    return var
```

Comment:
Shouldn't this be in ``utils.py`` why do you put it in the definition of the ``physio_obj.py``?

Reply:
Oh i realize now that is specific for the object, sry for the disturbance